tests/modules/nf-core/hisat2/build_test/main.nf

```diff
@@ -7,11 +7,32 @@ include { HISAT2_BUILD              } from '../../../../../modules/nf-core/hisat
 
 workflow test_hisat2_build {
     fasta = [ [id:'genome'],
-              file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+              file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
             ]
     gtf   = [ [id:'genome'],
-              file(params.test_data['sarscov2']['genome']['genome_gtf'], checkIfExists: true)
+              file(params.test_data['homo_sapiens']['genome']['genome_gtf'], checkIfExists: true)
```

Comment:
why changing from sarscov2 to homo_sapiens?

Reply:
ok, I read the comments, so makes sense

Reply:
all OK then?

Reply:
The sarscov2 annotation doesn't contain any exons to be extracted so the linting was complaining about having expected (intermediate) outputs that had empty md5sums. Indeed, this actually is a redundant change when I remove them from the `test.yml` as James advised. 

Should I revert back?

Reply:
No, it's fine to switch! Doesn't amke a difference

Reply:
Oh haha now I have switched back.